scripts/dash-scatterplot.py

```diff
@@ -0,0 +1,42 @@
+from dash import Dash, dcc, html, Input, Output
```

Comment:
We want a docstring here telling potential users and maintainers what this script does, with inputs and outputs. In a few words.

Reply:
file delete! just an example